README.md

```diff
@@ -33,56 +33,44 @@ List them by running the following command:
-                    The total memory is considered for this decision. 
-                    If this fails, or you actively overwrote the check with setting "--run_viewagg_on cpu", view agg is run on the cpu. 
-                    Equivalently, if you define "--run_viewagg_on gpu", view agg will be run on the gpu (no memory check will be done).
-* --device <str>: Select device for NN segmentation ("cpu", "cuda"), where cuda (default) means Nvidia GPU, you can select which one "cuda:1".
-* --batch: Batch size for inference. Default: 8. Lower this to reduce memory requirement
-* --order: Order of interpolation for mri_convert T1 before segmentation (0=nearest, 1=linear(default), 2=quadratic, 3=cubic)
-* --hires: Switch on hires processing (no conforming to 1mm, but to smallest voxel size).
-* --vol_segstats: Additionally return volume-based aparc.DKTatlas+aseg statistics for DL-based segmentation (does not  require surfaces). Can be used in combination with --seg_only in which case recon-surf only runs till CC is added (akin to --seg_with_cc_only).
+* `--fs_license`: Path to FreeSurfer license key file. Register (for free) at https://surfer.nmr.mgh.harvard.edu/registration.html to obtain it if you do not have FreeSurfer installed so far. Strictly necessary if you use Docker, optional for local install (your local FreeSurfer license will automatically be used). The license file is usually located in $FREESURFER_HOME/license.txt or $FREESURFER_HOME/.license .
+
+### Segmentation pipeline arguments (optional)
+* `--seg_only`: only run FastSurferCNN (generate segmentation, do not run the surface pipeline)
+* `--main_segfile`: Global path with filename of segmentation (where and under which name to store it). The file can be in mgz, nii, or nii.gz format. Default location: $SUBJECTS_DIR/$sid/mri/aparc.DKTatlas+aseg.deep.mgz
+* `--seg_log`: Name and location for the log-file for the segmentation (FastSurferCNN). Default: $SUBJECTS_DIR/$sid/scripts/deep-seg.log
+* `--viewagg_device`: Define where the view aggregation should be run on. Can be "auto" or a device (see --device). By default, the program checks if you have enough memory to run the view aggregation on the gpu. The total memory is considered for this decision. If this fails, or you actively overwrote the check with setting with "cpu" view agg is run on the cpu. Equivalently, if you pass a different device, view agg will be run on that device (no memory check will be done).
+* `--device`: Select device for NN segmentation ("cpu", "cuda"), where cuda means Nvidia GPU, you can select which one "cuda:1". Default: "auto", check GPU and then CPU
+* `--batch`: Batch size for inference. Default: 1. 
+* `--vol_segstats`: Additionally return volume-based aparc.DKTatlas+aseg statistics for DL-based segmentation (does not  require surfaces). Can be used in combination with `--seg_only` in which case recon-surf only runs till CC is added (akin to --seg_with_cc_only).
```

Comment:
`--seg_with_cc_only` does not sense any more. This option is removed.
Maybe this is also wrong in run_fastsurfer.sh

Reply:
This is resolved. See [here](https://github.com/Deep-MI/FastSurfer/pull/231)